cmd/apps/vpn-client/vpn-client.go

```diff
@@ -49,25 +50,29 @@ func main() {
 	if *serverPKStr == "" {
 		// TODO(darkrengarius): fix args passage for Windows
 		//*serverPKStr = "03e9019b3caa021dbee1c23e6295c6034ab4623aec50802fcfdd19764568e2958d"
-		log.Fatalln("VPN server pub key is missing")
+		fmt.Println("VPN server pub key is missing")
+		os.Exit(1)
 	}
 
 	serverPK := cipher.PubKey{}
 	if err := serverPK.UnmarshalText([]byte(*serverPKStr)); err != nil {
-		log.WithError(err).Fatalln("Invalid VPN server pub key")
+		fmt.Printf("Invalid VPN server pub key. error = %v", err)
```

Comment:
Could you please change the message to `Invalid VPN server pub key: %v` and apply the same format to other log messages? The format is `fmt.Printf("Error log message: %v", err)`.